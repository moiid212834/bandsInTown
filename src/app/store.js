import { configureStore } from '@reduxjs/toolkit';
import themeReducer from '../features/theme/ThemeSlice';
import artistReducer from '../features/artists/ArtistSlice';
import eventsReducer from '../features/events/EventsSlice';

/**
 * Configuring the store using redux toolkit and passing reducers
 */
export const store = configureStore({
  reducer: {
    theme: themeReducer,
    artists: artistReducer,
    events: eventsReducer,
  },
});
