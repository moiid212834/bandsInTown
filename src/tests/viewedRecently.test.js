import React from 'react';
import {Provider} from 'react-redux';
import {store} from '../app/store';
import {BrowserRouter as Router} from 'react-router-dom';
import renderer from 'react-test-renderer';
import ViewedRecently from '../components/viewedRecently';

it('renders correctly', () => {
    const tree = renderer
        .create(
        <Provider store={store}>
            <Router>
                <ViewedRecently></ViewedRecently>
            </Router>
        </Provider>
    )
        .toJSON();
    expect(tree).toMatchSnapshot();
});