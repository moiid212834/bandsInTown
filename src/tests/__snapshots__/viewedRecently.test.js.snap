// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h4
    className="MuiTypography-root MuiTypography-h4 css-pxw7rc-MuiTypography-root"
  >
    Recently Viewed
  </h4>
  <p>
    No Bands Recently Viewed
  </p>
</div>
`;
